docs/programming_guide/provisioning_system.rst

```diff
@@ -499,28 +499,7 @@ The following is an example of the default project.yml file of non-HA mode.
 
 .. _provision_command:
 
-*****************************
-Provision commandline options
-*****************************
-
-Running ``nvflare provision -h`` shows all available options.
-
-.. code-block:: shell
-
-    usage: nvflare provision [-h] [-p PROJECT_FILE] [-w WORKSPACE] [-c CUSTOM_FOLDER] [--add_user ADD_USER] [--add_client ADD_CLIENT]
-
-    optional arguments:
-    -h, --help                                               show this help message and exit
-    -p PROJECT_FILE, --project_file PROJECT_FILE                 file to describe FL project
-    -w WORKSPACE, --workspace WORKSPACE                          directory used by provision
-    -c CUSTOM_FOLDER, --custom_folder CUSTOM_FOLDER    additional folder to load python code
-    --add_user ADD_USER                                             yaml file for added user
-    --add_client ADD_CLIENT                                       yaml file for added client
-
-For options ``--add_user`` and ``--add_client``, please see the details in :ref:`dynamic_provisioning`.
-
-Running ``provision`` without any options and without a project.yml file in the current working directory will prompt
-to copy a default project.yml to the current working directory.
+.. include:: ../user_guide/nvflare_cli/provision_command.rst
```

Comment:
Question here:

Should we duplicate the text? (from user's perspective, I know we use include here to avoid duplication in rst file)

Or should we just provide a link that forward users to the page of "provision_command.rst"?